apps/native-ui-storybook/components/List/List.docs.mdx

```diff
@@ -0,0 +1,749 @@
+          trailingContent={
+            <Checkbox value="option1" aria-label="Label 1">
+              <CheckboxIndicator />
+            </Checkbox>
+          }
+          onPress={() => handlePress('option1')}
+        />
+        <ListItem
+          text="Mary Poppins"
+          supportingText="45 Somewhere Street"
+          leadingContent={<ListItemIcon as={UserMediumIcon} />}
+          trailingContent={
+            <Checkbox value="option2" aria-label="Label 2">
+              <CheckboxIndicator />
+            </Checkbox>
+          }
+          onPress={() => handlePress('option2')}
+        />
+        <ListItem
+          text="Charlie Brown"
+          supportingText="78 Somewhere Street"
+          leadingContent={<ListItemIcon as={UserMediumIcon} />}
+          trailingContent={
+            <Checkbox value="option3" aria-label="Label 3">
+              <CheckboxIndicator />
+            </Checkbox>
+          }
+          onPress={() => handlePress('option3')}
+        />
+        <ListItem
+          text="Jane Doe"
+          supportingText="90 Somewhere Street"
+          leadingContent={<ListItemIcon as={UserMediumIcon} />}
+          trailingContent={
+            <Checkbox value="option4" aria-label="Label 4">
+              <CheckboxIndicator />
+            </Checkbox>
+          }
+          onPress={() => handlePress('option4')}
+        />
+      </List>
+    </CheckboxGroup>
+  );
+};
+```
+
+For more information on the `Checbox` component, see the [`Checkbox` component documentation](/docs/native-ui-components-checkbox--docs).
+
+### Radio List
+
+You can also use the list to display a list of radio buttons. You can achieve this by using the `Checkbox` component as the `trailingContent` of the `ListItem`.
+
+<Canvas>
+  <NativeUIProvider>
+    <RadioList />
+  </NativeUIProvider>
+</Canvas>
+
+```tsx
+import {
+  List,
+  ListItem,
+  Radio,
+  RadioGroup,
+  RadioIndicator,
+  Image,
+  Box,
+  useColorMode,
+} from '@utilitywarehouse/native-ui';
+
+const RadioList = () => {
+  const [value, setValue] = React.useState<string>();
+  const colorMode = useColorMode();
+
+  const handlePress = (item: string) => {
+    setValue(item);
+  };
+  return (
+    <RadioGroup aria-label="Radio Group" nativeID="Radio-group" value={value}>
+      <List container="card">
+        <Box
+          borderRadius="$xl"
+          backgroundColor="$white"
+          $dark-backgroundColor="$darkGrey175"
+          overflow="hidden"
+        >
+          <ListItem
+            text="**** 5016"
+            supportingText="Expires 12/24"
+            divider
+            leadingContent={
+              <Image
+                source={{ uri: `/images/paymentMethods/visa/card${capitalize(colorMode)}.svg` }}
+                width={56}
+                height={42}
+              />
+            }
+            trailingContent={
+              <Radio value="option1" aria-label="Label 1">
+                <RadioIndicator />
```

Comment:
Not sure if this is a mistake 🤔 but the `RadioIndicator` seems to have a colour override when is `checked=true`

<img width="1705" alt="Screenshot 2024-08-27 at 13 41 08" src="https://github.com/user-attachments/assets/6d565899-4342-4444-855d-8c3b6e27f55e">

Initially I thought it was only in the context of List Item but it seems that the `RadioIndicator`  has this colour override as well

<img width="289" alt="Screenshot 2024-08-27 at 13 42 34" src="https://github.com/user-attachments/assets/ac1539a2-43f0-45c2-b5ef-12da2a7636d2">

Figma 👇 

<img width="551" alt="Screenshot 2024-08-27 at 13 43 57" src="https://github.com/user-attachments/assets/4f191e2a-b2bf-4221-96dc-a8dd667dc801">

Reply:
It's a weird issue with storybook, sometimes it builds like that for the web version. Not sure why. It will be resolved when we move away from Gluestack style and it doesn't affect the actual native output se
![Screenshot 2024-08-27 at 14 26 18](https://github.com/user-attachments/assets/e066c600-80ff-4c72-b913-2dbea9efcccb)
e the Apptetize output:

Reply:
Thanks for the feedback, I will use this view as well to QA